plugin/security/src/main/java/org/opencds/cqf/ruler/security/interceptor/AuthenticationInterceptor.java

```diff
@@ -21,6 +21,7 @@
 
 @Interceptor
 public class AuthenticationInterceptor implements org.opencds.cqf.ruler.api.Interceptor {
+	private static final String KUBERNETES_STARTUP_PROBE_PATH = "/fhir/metadata";
```

Comment:
This is just the metadata path. Nothing to do with Kubernetes. The healthcheck is performed the same for Docker Compose

Reply:
I searched and didn't see the path used anywhere but in helm and I was wanting to communicate context without comments.  

I'll update though.